Exercise_1.py

```diff
@@ -1,18 +1,30 @@
 class myStack:
      def __init__(self):
+          self.s = []
          
      def isEmpty(self):
+          if len(self.s) == 0:
+               return True 
+          else:
+               return False
          
      def push(self, item):
+          self.s.append(item)
```

Comment:
Do not use the inbuilt append and pop methods. Please try to implement it with custom pointers.